Userland/Demos/Starfield/Starfield.cpp

```diff
@@ -49,7 +49,7 @@ class Starfield final : public GUI::Widget {
 
     Vector<Coordinate> m_stars;
     int m_sweep_plane = 2000;
-    int m_speed = 1;
+    unsigned int m_speed = 1;
```

Comment:
Please only use `unsigned` for `unsigned int` (see the coding style)
Also, a suggestion - in case it goes unnoticed: https://github.com/SerenityOS/serenity/pull/10995#discussion_r753794486

Reply:
I tested with `unsigned speed = 1;` on `main()` and it automatically rejects the value if it's negative.
But thanks for the suggestion! :^)